Holovibes/includes/enum/enum_camera_kind.hh

```diff
@@ -25,5 +25,6 @@ enum class CameraKind
     AmetekS711EuresysCoaxlinkQSFP, /*!< Ametek S711 Euresys Coaxlink QSFP+ */
     Ametek,                        /*!< Ametek camera with EGrabber Studio */
     Alvium,                        /*!< Alvium-1800-u/2050 */
+    AutoDetectionPhantom,
```

Comment:
add comments. People before you were good people, they commented but you arrived and ruined everything